### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by BerrybeatSanz**
- Removed the test page from kte.tree
- Added the google slide for Coded's forms
- Moved my oc list to the moderator section and added Ult's stuff to their list cuz the placeholders presence was making my brain unhappy.

### CODE REVIEW

File: Writerside/kte.tree [hunk 1]
```diff
@@ -8,7 +8,6 @@
 
     <toc-element topic="starter-topic.md">
         <toc-element topic="Foundations.md">
-            <toc-element topic="BlueTest.md"/>
```
Comment (Interval0043) [inline Writerside/kte.tree:11]: looks great

File: Writerside/topics/Coded.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Weight
 
 ### Forms
 
-TBD
+[Forms Slide](https://docs.google.com/presentation/d/1FA_0Wax6TaCDa4aAaRdN3WsSzEx0_NW4CM26TPP0ppc/edit#slide=id.g31a71e86583_0_11)
```
Comment (Interval0043) [inline Writerside/topics/Coded.md:50]: we'll do this for now, but I'll figure out something soon

Comment (BerrybeatSanz) [reply Writerside/topics/Coded.md:50]: alr

File: Writerside/topics/OC-List.md
Comment (Interval0043) [inline Writerside/topics/OC-List.md:1]: tabs are good

**Comment by Interval0043** [comment]
here's the deal, i'm going to figure out how to enable cherry-picking in the future
in the meantime i'm going to hold on accepting until we get the tabs sorted out right away, even if it's a basic structure
i'm going to pour through documentation real fast

**Pull request merged**